pkg/parser/parser.peg

```diff
@@ -329,12 +329,15 @@ QuotedTextAttrs <- attrs:(QuotedTextRole / QuotedTextShortHand) {
 // This is for [role].  Anything between the brackets is
 // considered, but it must not start with a # or dot.  Anything after a comma
 // (including the comma) is discarded.  (Future support for named attributes?)
-QuotedTextRole <- "[" ![#.] role:QuotedTextRoleWord "]" {
-    return []interface{}{types.Attributes{ "role": role }}, nil
+QuotedTextRole <- "[" role:QuotedTextRoleWord? "]" {
+    //return []interface{}{types.Attributes{ "role": role }}, nil
```

Comment:
Probably we should drop the commented out text.

Reply:
done in 160be2b